src/System.IO.Packaging/src/project.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dependencies": {
+    "System.Collections": "4.0.10-beta-23008",
```

Comment:
I believe we're trying to avoid hardcoding revision numbers, e.g. these should end with "-*" instead of "-23008".  @ericstj can probably help if you run into issues.